src/View/User/Includes/_welcome_background_slideshow.html.twig

```diff
@@ -0,0 +1,10 @@
+{# User/Includes/_welcome_background_slideshow.html.twig #}
+
+<div class="slider">
+        <div class="load">
+        </div>
+        <div class="content">
+            <div class="principal">
+            </div>
+        </div>
```

Comment:
empty divs?

Reply:
these voids are provided for a card already created